### PR TITLE
fix hadolint Dockerfile scanning && CTK compilation failure

### PR DESCRIPTION
**Pull request by qiming-007**

### CODE REVIEW

File: .hadolint.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ignored:
+  - DL3008
```
Comment (Chen-Xintong) [inline .hadolint.yaml:2]: What is this file used for?

Comment (qiming-007) [reply .hadolint.yaml:2]: This is `hadolint` configuration file, see [hadolint configuration  ]( https://github.com/hadolint/hadolint#configure) for details. We can run `hadolint` quickly in the future with the configuration for `Dockerfile` scanning.

Comment (Chen-Xintong) [reply .hadolint.yaml:2]: Got it, thanks!

**Review by Chen-Xintong — APPROVED** [review]
LGTM

**Pull request merged**